src/Microsoft.Extensions.Options/IConfigureNamedOptions.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Options
+{
+
+    /// <summary>
+    /// Represents something that configures the TOptions type.
+    /// </summary>
+    /// <typeparam name="TOptions"></typeparam>
+    public interface IConfigureNamedOptions<in TOptions> where TOptions : class
```

Comment:
Old API's will do (behave) the same, but only in a bit different way. End users won't notice the different, but they will know that named and unnamed options work alike.

Yes, I can write a helper method that will call `Configure` for each name that I need (in cycle call `Configure` for each name with the same action). But having something like: `Configure(["name1", "name2"], o => DoStuff())` could help reduce the count of total `IConfigureOptions` instances (and because we iterate through all of them - potentially reduce the time).